snapshots/un/2023-10-02/un_wpp_lt_all.zip.dvc

```diff
@@ -0,0 +1,39 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: World Population Prospects
+    description: |-
+      World Population Prospects 2022 is the 27th edition of the official global population estimates and projections published by the United Nations since 1951. The estimates are based on all available data sources on population size and fertility levels, mortality, and international migration for 237 countries or areas. 
+
+      For each revision, any new, recent but also historical, information that has become available from population censuses, vital registration of births and deaths, and household surveys is considered to produce consistent time series of population estimates for each country or area from 1950 to today. 
+
+      For the estimation period between 1950 and 2022, data from 1,758 censuses were considered in the present evaluation. In some countries, population registers based on administrative data systems provide the necessary information. Population data from censuses or registers referring to 2015 or later were available for 152 countries or areas, representing 64 per cent of the 237 countries or areas included in this analysis. For 74 countries or areas, the most recent available population count was from 2005-2014. For the remaining 11 countries or areas, the most recent available census data were from before 2005. In addition, information on births and deaths from civil registration and vital statistics systems for 169 countries, and demographic indicators from 2,890 surveys were considered in the present evaluation.
+    date_published: 2022-07-11
+    version_producer: 2022
+    title_snapshot: World Population Prospects - Life Tables (Both sexes)
+    description_snapshot: |-
+      Provides single-age life tables up to age 100 for both sexes with a set of values showing the mortality experience of a hypothetical group of infants born at the same time and subject throughout their lifetime to the specific mortality rates of a given year. The following series are provided: age-specific mortality rates (mx), probabilities of dying (qx), probabilities of surviving (px), number surviving (lx), number dying (dx), number of person-years lived (Lx), survivorship ratios (Sx), cumulative stationary population (Tx), average remaining life expectancy (ex) and average number of years lived (ax).
+
+    # Citation
+    producer: United Nations
+    citation_full: |-
+      United Nations, Department of Economic and Social Affairs, Population Division (2022). World Population Prospects 2022, Online Edition.
+    attribution: United Nations - World Population Prospects 2022 (2022)
```

Comment:
Maybe the double 2022 is not needed?

Reply:
Yeah, not great.

I copied it from their suggested citation ([see the footer](https://population.un.org/wpp/Download/Standard/MostUsed/)), so I'd leave it as is for now.